servicecontrol/containerization/index.md

```diff
@@ -16,32 +16,34 @@ ServiceControl is split into multiple docker images for each instance type. Thes
 | Azure Container Services (ACS)          | No        | Works, but does not support persistent volumes for durable storage |
 | Azure Kubernetes Services (AKS)         | No        | Supports minimum of Windows Server 2019-based images               |
 | Amazon Elastic Container Services (ECS) | No        | Untested, guidance mentions AMIs available for Windows 2016        |
 | Amazon Elastic Kubernetes Service (EKS) | No        | EKS only supports Windows 2019                                     |
 
+Note: There is work in progress to enable ServiceControl to run in Linux containers. The details of the work are available on the [GitHub issue](https://github.com/Particular/ServiceControl/issues/3651).
+
 ## Init containers
 
-The init containers are used to create or upgrade the infrastructure required for ServiceControl and are based on the [Kubernetes init containers](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/). Once the init container has been run, it will shut down and the runtime container can be run. The runtime container will use the infrastructure that has been created and if the init container was not run the runtime container will fail.
+The `init` containers are used to create or upgrade the infrastructure required for ServiceControl. They are based on the [Kubernetes `init` containers](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/). Once the `init` container has been run, it will shut down and the `runtime` container can be run. The `runtime` container uses the infrastructure that has been created by the `init` container and will fail if the `init` container is not run.
 
-The init containers are similar to [installing all ServiceControl service types (regular, error, and monitoring) via the ServiceControl Management Utility or the Powershell Cmdlets](/servicecontrol/installation.md). This includes creating the database as well as the queues that ServiceControl uses and runtime containers are similar to starting the corresponding Windows Services.
+The `init` containers are similar to [installing all ServiceControl service types (regular, error, and monitoring) via the ServiceControl Management Utility or the Powershell Cmdlets](/servicecontrol/installation.md). This includes creating the database and the queues that ServiceControl uses. The `runtime` containers are similar to starting the corresponding Windows Services.
 
-Init containers are identical to the corresponding runtime image but with an added entrypoint arguments (`--setup`). It is also possible to use the runtime image and override the entrypoint and add this argument to achieve the same goal.
+`Init` containers are identical to the corresponding `runtime` image but with added entry point arguments (`--setup`). It is also possible to use the `runtime` image, override the entry point and add this argument to achieve the same goal.
```

Comment:
Can we provide an example to enhance this?

Reply:
It sounds like an implementation detail. I'm leaning toward removing it.

Reply:
There is something similar in line 218 on this documentation which is what got me thinking